src/Compilers/CSharp/Test/Syntax/Parsing/ParserRegressionTests.cs

```diff
@@ -190,11 +190,27 @@ public int
             ParseAndValidate(test,
                 // (6,1): error CS1519: Invalid token '}' in class, record, struct, or interface member declaration
                 // }
-                Diagnostic(ErrorCode.ERR_InvalidMemberDecl, "}").WithArguments("}").WithLocation(6, 1),
-                // (4,16): warning CS0078: The 'l' suffix is easily confused with the digit '1' -- use 'L' for clarity
-                //     [Obsolete(2l)]
-                Diagnostic(ErrorCode.WRN_LowercaseEllSuffix, "l").WithLocation(4, 16)
-                );
+                Diagnostic(ErrorCode.ERR_InvalidMemberDecl, "}").WithArguments("}").WithLocation(6, 1));
+        }
+
+        [Fact, WorkItem(13719, "https://github.com/dotnet/roslyn/issues/13719")]
+        public void ReportErrorForIncompleteMember2()
+        {
+            // This test demonstrates a current parser bug.  We do not report errors on incomplete members if there is
+            // already a warning diagnostics on it.
```

Comment:
I don't understand this comment? We _are_ reporting an error?

Reply:
indeed.  the comment is wrong.  i originally added the test (which was behaving incorrectly).  I then fixed the parser.  And i didn't remove the comment :)